packages/math-input/src/types.ts

```diff
@@ -9,9 +9,18 @@ import type {Key} from "./data/keys";
 
 export type Border = Partial<Array<keyof typeof BorderDirections>>;
 
+export type Bound = {
+    top: number;
+    right: number;
+    bottom: number;
+    left: number;
+    height: number;
+    width: number;
+};
+
 export type Popover = {
     parentId: Key;
-    bounds: DOMRect;
+    bounds: Partial<Bound>;
```

Comment:
IIRC the original devs were taking this object and dumping it into styles, but DOMRect includes things that can't be added to styles. Since I'm trying not to change logic, I decided to just make a type with only the fields needed.